README.md

```diff
@@ -148,6 +148,29 @@ from binding to a local port or blocking communication with Cloud IAP.
 
 To build the plugin, check out the [prerequisites](BUILDING.md).
 
+## Privacy
+
+This plugin accesses Google Cloud Platform in order to:
+
+* establish Cloud IAP TCP tunnels to VM instances
+* list VMs and obtain metadata and logs for VM instances  
+* generate Windows logon credentials if requested
+
+The plugin uses your local installation of [gcloud](https://cloud.google.com/sdk/gcloud/) 
+as well as the following APIs for this purpose:
+
+* [Compute Engine API](https://cloud.google.com/compute/docs/reference/rest/v1/)
+* [Google OAuth](https://developers.google.com/identity/protocols/OAuth2)
+
+Authentication is performed using your locally saved gcloud credentials.
+
+The plugin does not disclose or transmit any user data to APIs other than the
```

Comment:
nor does gcloud either 

https://cloud.google.com/sdk/usage-statistics

if this is boilerplate words, then its fine.

if its not, then maybe some more stuff
eg. how can someone witpeout creds, etc
fwiw, only the compute api is consumtion based (the oauth is free though)

Reply:
It is not exactly boilerplate words, but I have say a few things about privacy to get OAuth approval.

Added links to gcloud docs.